src/org/zaproxy/zap/extension/pscan/ExtensionPassiveScan.java

```diff
@@ -548,6 +554,16 @@ public void loadFrom(Configuration conf) {
         }
 	}
 
+    public ScanStatus getScanStatus() {
```

Comment:
- `protected`.
 - The counter should be reset when the session changes (otherwise it might show an outdated number, from previous session).